admin/src/main/resources/io/buoyant/admin/js/src/dashboard.js

```diff
@@ -2,18 +2,20 @@
 
 define([
   'jQuery', 'bootstrap',
+  'src/router_services',
   'src/metrics_collector',
   'src/process_info',
   'src/request_totals',
   'src/router_controller'
 ], function(
   $, bootstrap,
+  RouterServices,
   MetricsCollector,
   ProcInfo,
   RequestTotals,
   RouterController
 ) {
-  return function(routerConfig) {
+  return function(routerConfig, isServiceMap) {
```

Comment:
Yeah, I think `isServiceMap` was also a bad arg name :) I originally called the service dashboard "serviceMap" but forgot to rename it in all places.

Would it be less confusing if this argument was called `dashboardType` instead?

Renaming `RouterController` to `ClientDashboard` and `RouterServices` to `ServiceDashboard` could also help. (They both need the same setup things, e.g. initializing ProcInfo and RequestTotals)

Then line 43 would be something like:
```
if(dashboardType === "service") {
  ServiceDashboard(metricsCollector, initialData, $(".service-dashboard-container"));
} else {
  ClientDashboard(metricsCollector, selectedRouter, initialData, $(".client-dashboard-container"), routerConfig);
}
```

This file could be called something better, like `dashboard_initializer` which is maybe more indicative of what it is doing.